Technical error with connectivity (?)

Not sure what exactly caused it, but I just updated the app to latest dev version (via F-Droid), so I guess it is related to that:

```
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Error receiving broadcast Intent { act=android.net.wifi.STATE_CHANGE flg=0x4000010 (has extras) } in com.owncloud.android.utils.ReceiversHelper$1@ee07b69
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$-android_app_LoadedApk$ReceiverDispatcher$Args_52511(LoadedApk.java:1323)
	at android.app.-$Lambda$aS31cHIhRx41653CMnd4gZqshIQ.$m$7(Unknown Source:4)
	at android.app.-$Lambda$aS31cHIhRx41653CMnd4gZqshIQ.run(Unknown Source:39)
	at android.os.Handler.handleCallback(Handler.java:790)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6494)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.net.NetworkCapabilities.hasCapability(int)' on a null object reference
	at com.nextcloud.client.network.ConnectivityServiceImpl.getConnectivity(ConnectivityServiceImpl.java:121)
	at com.owncloud.android.utils.ReceiversHelper$1.onReceive(ReceiversHelper.java:58)
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$-android_app_LoadedApk$ReceiverDispatcher$Args_52511(LoadedApk.java:1313)
	... 9 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20201127
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: Fairphone
Device: FP2
Model: FP2
Id: OPM7.181105.004
Product: FP2

************ FIRMWARE ************
SDK: 27
Release: 8.1.0
Incremental: de8920e892

```

--- a/src/main/java/com/nextcloud/client/network/ConnectivityServiceImpl.java
+++ b/src/main/java/com/nextcloud/client/network/ConnectivityServiceImpl.java
@@ -118,7 +118,11 @@ class ConnectivityServiceImpl implements ConnectivityService {
                 NetworkCapabilities networkCapabilities = platformConnectivityManager.getNetworkCapabilities(
                     platformConnectivityManager.getActiveNetwork());
 
-                isMetered = !networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_NOT_RESTRICTED);
+                if (networkCapabilities != null) {
+                    isMetered = !networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_NOT_RESTRICTED);
+                } else {
+                    isMetered = ConnectivityManagerCompat.isActiveNetworkMetered(platformConnectivityManager);
+                }
             } else {
                 isMetered = ConnectivityManagerCompat.isActiveNetworkMetered(platformConnectivityManager);
             }

